Fix read/write policy outputs

dornera opened the pull request:
* Correctly outputs/converts read/write policy from a list to a string
* Will need to fix any references to this module and remove [0] from them

gravesm commented:
I don't understand why we appear to have multiple of each policy. Shouldn't there just be one policy?

dornera committed:
Updated to auto create policies

dornera commented:
Ok as we discussed and extras
* Create policies by default
* Cleaned up extra splat syntax in outputs (since we don't use enabled)
* Removed `access` var
